release_notes.md

```diff
@@ -5,3 +5,40 @@
 The file describes all the changes, that are affected the usage of the application.
 
 The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).
+
+[0.1.0] - [26.05.2023]
+
+### New features
+
+`Quicklook`:  Users can check a list of pet sitters available for pet owners. Each row of the list includes an avatar, name + surname, price, short bio and rating of pet sitters.
+
+</details>
+
+<details>
+<summary> CLICK HERE</summary>
+
+<p>
+
+![list of pet sitters](https://github.com/ios-course/ironfoudation-team-project/assets/120586844/a62ea90f-50f1-4c44-81c1-e0ef8091feae)
+
+</p>
+</details>
+
+`Remote check the pet sitter`: App allows users without internet connection tab to the pet sitters' row in the list of sitters and read more detailed information about the sitter:  pet sitter's phone number, rating and full bio.
+
+</details>
+
+<details>
+<summary> CLICK HERE</summary>
+
+<p>
+
+![sitter](https://github.com/ios-course/ironfoudation-team-project/assets/120586844/4994d3ef-8b12-4083-8924-1730c9b53ef6)
+
+</p>
+</details>
+
+`Easy to navigate`:  Users can easily switch between pet sitters' list and the pet sitter window with detailed information.
```

Comment:
Does it make sense to mention this?
It's standard navigation.

Reply:
Yes, sure. The line are deleted.